x-pack/test/apm_api_integration/tests/transactions/error_rate.spec.ts

```diff
@@ -91,149 +77,185 @@ export default function ApiTest({ getService }: FtrProviderContext) {
+                  )
+                  .duration(1000)
+                  .timestamp(timestamp)
+                  .failure()
+              )
+          );
+      });
 
-        it('returns some data', () => {
-          expect(errorRateResponse.currentPeriod.average).to.be.greaterThan(0);
-          expect(errorRateResponse.previousPeriod.average).to.be(null);
+      await synthtraceEsClient.index(documents);
+    });
 
-          expect(errorRateResponse.currentPeriod.timeseries.length).to.be.greaterThan(0);
-          expect(errorRateResponse.previousPeriod.timeseries).to.empty();
+    after(() => synthtraceEsClient.clean());
 
-          const nonNullDataPoints = errorRateResponse.currentPeriod.timeseries.filter(
-            ({ y }) => y !== null
-          );
+    describe('returns the transaction error rate', () => {
+      let errorRateResponse: ErrorRate;
 
-          expect(nonNullDataPoints.length).to.be.greaterThan(0);
+      before(async () => {
+        const response = await fetchErrorCharts({
+          query: { transactionName: config.firstTransaction.name },
         });
+        errorRateResponse = response.body;
+      });
 
-        it('has the correct start date', () => {
-          expectSnapshot(
-            new Date(first(errorRateResponse.currentPeriod.timeseries)?.x ?? NaN).toISOString()
-          ).toMatchInline(`"2021-08-03T06:50:00.000Z"`);
-        });
+      it('returns some data', () => {
+        expect(errorRateResponse.currentPeriod.average).to.be.greaterThan(0);
+        expect(errorRateResponse.previousPeriod.average).to.be(null);
 
-        it('has the correct end date', () => {
-          expectSnapshot(
-            new Date(last(errorRateResponse.currentPeriod.timeseries)?.x ?? NaN).toISOString()
-          ).toMatchInline(`"2021-08-03T07:20:00.000Z"`);
-        });
+        expect(errorRateResponse.currentPeriod.timeseries.length).to.be.greaterThan(0);
+        expect(errorRateResponse.previousPeriod.timeseries).to.empty();
 
-        it('has the correct number of buckets', () => {
-          expectSnapshot(errorRateResponse.currentPeriod.timeseries.length).toMatchInline(`31`);
-        });
+        const nonNullDataPoints = errorRateResponse.currentPeriod.timeseries.filter(
+          ({ y }) => y !== null
+        );
 
-        it('has the correct calculation for average', () => {
-          expectSnapshot(errorRateResponse.currentPeriod.average).toMatchInline(
-            `0.0848214285714286`
-          );
-        });
+        expect(nonNullDataPoints.length).to.be.greaterThan(0);
+      });
 
-        it('has the correct error rate', () => {
-          expectSnapshot(errorRateResponse.currentPeriod.timeseries).toMatch();
-        });
+      it('has the correct start date', () => {
+        expectSnapshot(
+          new Date(first(errorRateResponse.currentPeriod.timeseries)?.x ?? NaN).toISOString()
+        ).toMatchInline(`"2021-01-01T00:00:00.000Z"`);
```

Comment:
I'm not sure snapshots are that valuable anymore. We can keep it around for now but we don't have to add more snapshot assertions